packages/ember-metal/lib/chains.js

```diff
@@ -10,12 +10,17 @@ function firstKey(path) {
   return path.match(FIRST_KEY)[0];
 }
 
-function isObject(obj) {
-  return obj && (typeof obj === 'object');
+function isWatchable(obj) {
```

Comment:
I did until async dependency undefineds made me want to cry to the point of sneaking this in. I can take a stab at it again when I emotionally recover.